Patch created by running brand_python.py on Python 3.5.1 source
diff --git Lib/platform.py Lib/platform.py
index f3d2c42..7e42c78 100755
--- Lib/platform.py
+++ Lib/platform.py
@@ -1148,7 +1148,7 @@ def processor():
 ### Various APIs for extracting information from sys.version
 
 _sys_version_parser = re.compile(
-    r'([\w.+]+)\s*'
+    r'([\w.+]+)\s*\ \|\ packaged\ by\ conda\-forge\ \|\s*'
     '\(#?([^,]+),\s*([\w ]+),\s*([\w :]+)\)\s*'
     '\[([^\]]+)\]?', re.ASCII)
 
diff --git Python/getversion.c Python/getversion.c
index 7bd6efd..998c4b3 100644
--- Python/getversion.c
+++ Python/getversion.c
@@ -9,7 +9,8 @@ const char *
 Py_GetVersion(void)
 {
 	static char version[250];
-	PyOS_snprintf(version, sizeof(version), "%.80s (%.80s) %.80s",
+    PyOS_snprintf(version, sizeof(version),
+        "%.80s | packaged by conda-forge | (%.80s) %.80s",
 		      PY_VERSION, Py_GetBuildInfo(), Py_GetCompiler());
 	return version;
 }
-- 
2.7.2

